config/routes.rb

```diff
@@ -8,9 +8,11 @@
 
```

Comment:
I am not sure about the routes.
@lxingern what do you think?
WHy is create nested and destroy is not?

Reply:
verified that destroy is not to be nested. same for create